src/client/components/forms/userCollection.js

```diff
@@ -0,0 +1,306 @@
+		}));
+	}
+
+	handleRemoveCollaborator(index) {
+		const newCollaborators = this.state.collection.collaborators;
+		newCollaborators.splice(index, 1);
+		this.setState(prevState => ({
+			collection: {
+				...prevState.collection,
+				collaborators: newCollaborators
+			},
+			errorText: prevState.errorText
+		}));
+	}
+
+	handleChangeCollaborator(newCollab, index) {
+		let newCollaborator;
+		if (!newCollab) {
+			newCollaborator = {
+				id: null,
+				name: ''
+			};
+		}
+		else {
+			newCollaborator = newCollab;
+		}
+		const newCollaborators = this.state.collection.collaborators;
+		newCollaborators[index] = newCollaborator;
+		this.setState(prevState => ({
+			collection: {
+				...prevState.collection,
+				collaborators: newCollaborators
+			},
+			errorText: prevState.errorText
+		}));
+	}
+
+	render() {
+		if (this.state.collection.collaborators.length === 0) {
+			this.handleAddCollaborator();
+		}
+
+		const privacyOptions = ['Private', 'Public'].map((option) => ({
+			name: option
+		}));
+		const entityTypeOptions = ['Author', 'Work', 'Edition', 'Edition-Group', 'Publisher'].map((entity) => ({
+			name: entity
+		}));
+		const initialName = this.state.collection.name;
+		const initialDescription = this.state.collection.description;
+		const initialPrivacy = this.state.collection.public ? 'Public' : 'Private';
+		const initialType = this.state.collection.entityType;
+		const {errorText} = this.state;
+		const errorAlertClass =
+			classNames('text-center', 'margin-top-1', {hidden: !errorText});
+		const submitLabel = this.props.collection.name ? 'UPDATE' : 'CREATE';
+
+		/* eslint-disable react/jsx-no-bind */
+		return (
+			<Grid>
+				<h1>Create a collection</h1>
+				<Row>
+					<Col md={12}>
+						<p className="lead">Create Your Collection</p>
+					</Col>
+				</Row>
+				<Row>
+					<Col
+						id="collectionForm"
+						md={8}
+						mdOffset={2}
+					>
+						<form
+							className="form-horizontal"
+							onSubmit={this.handleSubmit}
+						>
+							<CustomInput
+								defaultValue={initialName}
+								label="Name"
+								ref={(ref) => this.name = ref}
+								type="text"
+							/>
+							<CustomInput
+								defaultValue={initialDescription}
+								label="Description"
+								ref={(ref) => this.description = ref}
+								type="textarea"
+							/>
+							<SelectWrapper
+								base={ReactSelect}
+								defaultValue={initialType}
+								disabled={!this.props.canEditType}
+								idAttribute="name"
+								label="Entity Type"
+								labelAttribute="name"
+								options={entityTypeOptions}
+								placeholder="Select Entity Type"
+								ref={(ref) => this.entityType = ref}
+							/>
+							<SelectWrapper
+								base={ReactSelect}
+								defaultValue={initialPrivacy}
+								idAttribute="name"
+								label="Privacy"
+								labelAttribute="name"
+								options={privacyOptions}
+								placeholder="Select Privacy"
+								ref={(ref) => this.privacy = ref}
+							/>
+							{
```

Comment:
You could add a small separation between the collection details and the collaborators section, like an `<hr>` element, and add a small title for that section.

Reply:
I still think a separation between the main body of the form and the collaborators will be helpful.